sql/core/src/main/scala/org/apache/spark/sql/ContinuousQueryManager.scala

```diff
@@ -175,9 +175,9 @@ class ContinuousQueryManager(sparkSession: SparkSession) {
       checkpointLocation: String,
       df: DataFrame,
       sink: Sink,
+      outputMode: OutputMode,
       trigger: Trigger = ProcessingTime(0),
-      triggerClock: Clock = new SystemClock(),
-      outputMode: OutputMode = Append): ContinuousQuery = {
+      triggerClock: Clock = new SystemClock()): ContinuousQuery = {
```

Comment:
I just realized we should make the constructor of `ContinuousQueryManager` be `private[sql]` since we don't want people to call it. Could you also fix it in this PR?